.gitignore

```
@@ -140,6 +140,11 @@
 **/vendor/bundle/ruby/*/gems/unicode-display_width-*/
 **/vendor/bundle/ruby/*/gems/webrobots-*/
 
+# optional gems
+**/vendor/bundle/ruby/*/gems/bindata-*/
+**/vendor/bundle/ruby/*/gems/elftools-*/
+**/vendor/bundle/ruby/*/gems/patchelf-*/
```

```suggestion
**/vendor/bundle/ruby/*/gems/bindata-*/
**/vendor/bundle/ruby/*/gems/elftools-*/
**/vendor/bundle/ruby/*/gems/patchelf-*/
```

Please add these to `# Ignore dependencies we don't wish to vendor` above or move `sorbet`, `sorbet-runtime` and `tapioca` here and be something like `# Ignore gems conditional on environment variables` for the title